src/server.rs

```diff
@@ -62,6 +62,22 @@ pub extern "C" fn rustls_server_config_builder_new() -> *mut rustls_server_confi
     }
 }
 
+/// With `ignore` != 0, the server will ignore the client ordering of cipher
+/// suites, aka preference, during handshake and respect its own ordering
+/// as configured.
+/// https://docs.rs/rustls/0.19.0/rustls/struct.ServerConfig.html#fields
+#[no_mangle]
+pub extern "C" fn rustls_server_config_builder_set_ignore_client_order(
+    builder: *mut rustls_server_config_builder,
+    ignore: i8,
```

Comment:
The field on `ServerConfig` is `bool`, so why not plumb a `bool` all the way to the C interface? We already return `bool` in a few places so I don't think there's a concern about `stdbool.h` not being available.